src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -550,26 +550,22 @@ char *wm_vuldet_build_url(char *pattern, char *value) {
 }
 
 int wm_vuldet_compare_pkg(cve_vuln_pkg *Pkg1, cve_vuln_pkg *Pkg2) {
-    if (!(!Pkg1->src_name && !Pkg2->src_name) &&
-        !(Pkg1->src_name && Pkg2->src_name &&
+    if (!(Pkg1->src_name && Pkg2->src_name &&
```

Comment:
@TomasTurina wouldn't be enough just doing this check?

Reply:
The problem will be with RedHat systems, where `src_name` is always NULL. Some packages of Ubuntu and Debian also have `src_name` NULL. That condition isn't enought, since if `src_name` is NULL then the function will assume that the package is different but it could be the same.

Reply:
Thanks for the explanation! I supposed that should have a reason. So, restoring the original conditions.